web/modules/custom/nys_registration/src/Form/RegisterForm.php

```diff
@@ -381,9 +336,9 @@ public function formBuildStep2(array &$form, FormStateInterface $form_state): ar
       $fid = $media->get('field_image')->target_id;
       $file = File::load($fid);
       $senator['image'] = empty($file) ?
-            '/themes/custom/nysenate_theme/src/assets/default-avatar.png' :
-            \Drupal::service('file_url_generator')
-              ->generateAbsoluteString($file->getFileUri());
+        '/themes/custom/nysenate_theme/src/assets/default-avatar.png' :
```

Comment:
This is the preferred way to get the path to a theme. You could take it a bit further if you don't want to hardcode the theme name (probably not worth it).

`$default_theme_name = \Drupal::config('system.theme')->get('default');`

This might be worth it though.

` $theme_path = \Drupal::service('extension.list.theme')->getPath('nysenate_theme');
    $default_avatar = '/' .  $theme_path . '/src/assets/default-avatar.png';
    $senator['image'] = empty($file) ? $default_avatar;
`

Reply:
This is the way, but it belongs in a theme-related helper service.  That bit of overengineering will have to wait.  :)